core/assets/bundles/bundle_de.properties

```diff
@@ -963,9 +963,9 @@ ability.forcefield = Kraftfeld
 ability.repairfield = Heilungsfeld
 ability.statusfield = {0} Statusfeld
 ability.unitspawn = {0} Fabrik
-ability.shieldregenfield = Schild-regenerations-Feld
+ability.shieldregenfield = Schildregenerationsfeld
 ability.movelightning = Bewegungsblitze
-ability.shieldarc = Lichtbogenschild
+ability.shieldarc = Schildbogen
```

Comment:
I'd want to stay with 'Lichtbogen' for clarity and uniformity personally - I'd also prefer 'Lichtbogenschild' over 'Schildlichtbogen' as we're referring to an arc-shaped shield and not an arc that also happens to be a shield. Open to other ideas though.

Reply:
Sure, I'm fine with the old order too, it was more of an observation anyway